tests/multi-platform/multi-platform.go

```diff
@@ -0,0 +1,236 @@
+		)
+
+		componentName = fmt.Sprintf("multi-platform-suite-component-%s", util.GenerateRandomString(4))
+
+		// Create a component with Git Source URL being defined
+		componentObj := appservice.ComponentSpec{
+			ComponentName: componentName,
+			Source: appservice.ComponentSource{
+				ComponentSourceUnion: appservice.ComponentSourceUnion{
+					GitSource: &appservice.GitSource{
+						URL:      testProjectGitUrl,
+						Revision: testProjectRevision,
+					},
+				},
+			},
+		}
+		component, err = f.AsKubeAdmin.HasController.CreateComponent(componentObj, testNamespace, "", "", applicationName, true, map[string]string{})
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+
+	When("the Component with multi-platform-build is created", func() {
+		It("a PipelineRun is triggered", func() {
+			Eventually(func() error {
+				pr, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(componentName, applicationName, testNamespace, "")
+				if err != nil {
+					GinkgoWriter.Printf("PipelineRun has not been created yet for the component %s/%s", testNamespace, componentName)
+					return err
+				}
+				if !pr.HasStarted() {
+					return fmt.Errorf("pipelinerun %s/%s hasn't started yet", pr.GetNamespace(), pr.GetName())
+				}
+				return nil
+			}, timeout, constants.PipelineRunPollingInterval).Should(Succeed(), fmt.Sprintf("timed out when waiting for the PipelineRun to start for the component %s/%s", testNamespace, componentName))
+		})
+
+		It("the build-container task from component pipelinerun is buildah-remote", func() {
+
+			Eventually(func() error {
+				pr, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(componentName, applicationName, testNamespace, "")
+				Expect(err).ShouldNot(HaveOccurred())
+
+				for _, chr := range pr.Status.ChildReferences {
+					taskRun := &v1beta1.TaskRun{}
+					taskRunKey := types.NamespacedName{Namespace: pr.Namespace, Name: chr.Name}
+					err := f.AsKubeAdmin.CommonController.KubeRest().Get(context.TODO(), taskRunKey, taskRun)
+					Expect(err).ShouldNot(HaveOccurred())
+
+					prTrStatus := &v1beta1.PipelineRunTaskRunStatus{
+						PipelineTaskName: chr.PipelineTaskName,
+						Status:           &taskRun.Status,
+					}
+
+					if chr.PipelineTaskName == constants.BuildTaskRunName && prTrStatus.Status != nil && prTrStatus.Status.TaskSpec != nil && prTrStatus.Status.TaskSpec.Volumes != nil {
+						for _, vol := range prTrStatus.Status.TaskSpec.Volumes {
+							if vol.Secret != nil && strings.HasPrefix(vol.Secret.SecretName, "multi-platform-ssh-") {
+								return nil
+							}
+						}
+					}
+				}
+				return fmt.Errorf("couldn't find a matching step buildah-remote in task %s in PipelineRun %s/%s", constants.BuildTaskRunName, testNamespace, pr.GetName())
+			}, timeout, interval).Should(Succeed(), "timed out when verifying the request processor image reference in pipelinerun")
```

Comment:
should we change the message in the `Should` block here .i.e. `timed out when verifying the request processor image reference in pipelinerun` ?